.travis.yml

```diff
@@ -90,6 +90,18 @@ matrix:
     - stage: test
       python: 3.6
       env: TOX_ENV=py36-extra-lib
+    - stage: test
+      python: 3.7
+      env: TOX_ENV=py37-oldest-lib
+    - stage: test
+      python: 3.7
+      env: TOX_ENV=py37-pinned-lib
+    - stage: quick_test3
```

Comment:
I think the idea behind the `quick_test` is that there's one variant on each Python version that runs first. So we should drop either this one or the python 3.6 one. Probably best to keep the latest version

Reply:
changed all the "single py2/py3" tests to only use py37